albumentations/augmentations/transforms.py

```diff
@@ -921,11 +927,20 @@ class Cutout(ImageOnlyTransform):
 
     """
 
-    def __init__(self, num_holes=8, max_h_size=8, max_w_size=8, always_apply=False, p=0.5):
+    def __init__(self, max_holes=8, max_height=8, max_width=8,
+                 min_holes=None, min_height=None, min_width=None,
+                 always_apply=False, p=0.5):
         super(Cutout, self).__init__(always_apply, p)
-        self.num_holes = num_holes
-        self.max_h_size = max_h_size
-        self.max_w_size = max_w_size
+        self.max_holes = max_holes
+        self.max_height = max_height
+        self.max_width = max_width
+        self.min_holes = min_holes if min_holes is not None else max_holes
+        self.min_height = min_height if min_height is not None else max_height
+        self.min_width = min_width if min_width is not None else max_width
+
+        assert 0 < self.min_holes and self.min_holes <= self.max_holes
```

Comment:
I would believe that 

`assert 0 < self.min_holes <= self.max_holes`

is a better syntax.

Reply:
Fair point. I guess the pain from using C++ where it would evaluate into `((0 < self.min_holes) <= self.max_holes)` is somehow hardwired in my brain so I decided to use this instead.